ratchet_core/src/split/mod.rs

```diff
@@ -530,6 +536,37 @@ where
     }
 }
 
+impl<S> WebSocketClose for WriteHalf<S>
+where
+    S: WebSocketStream,
+{
+    fn write(&mut self, code: CloseCode) -> BoxFuture<Result<(), Error>> {
```

Comment:
Slight niggle: this should propbably be something like `write_close_frame` as it's not a general `write` method.